moe.nimble

```diff
@@ -1,6 +1,6 @@
 # Package
 
-version       = "0.4.0"
+version       = "0.3.0"
```

Comment:
Yes, we need to do so.  That is the somehow confusing thing about bump2version:  when it returns, the version is incremented.  bump2version assumes the version to be updated ***right before*** the new release is published instead of directly after publishing it.

Reply:
@fox0430

This is the example:  https://github.com/kevinmatthes/git-aliases-f18/pull/95.

The repository uses the very same workflow as I suggest here; the changes I submit with this PR are copied from there (which is no problem since it the repository is one of my own repositories).

I released the software in that repository with the version v0.1.3.  Let us assume that after some work, I can now release v0.2.0.  Please look at the changed files; https://github.com/kevinmatthes/git-aliases-f18/pull/95/files.  The whole time I worked on the changes for the new version, I still used the old version number.  I only increment it on release.  This requires that all tracked files have the same version number hard-coded somewhere.

This is why I needed to downgrade the version in `moe.nimble`:  because it will be re-upgraded on release.

(The example is also a preview of how a release with the suggested workflow will look like.  Please ignore the CI in the example; it cannot fetch Valgrind for memory integrity check and thus takes that long time before it will, probably, fail.)

Reply:
@kevinmatthes 

Thank you for the example. I understand ti.